test_aggregation_different_types_crate random failures

**Describe the bug**
`test_aggregation_different_types_crate` is failing randomly, most probably due to cratedb consolidation.
while we fixed the issue for old test, this was recently merged, and probably is not using the correct
wait_for mechanism.

```
AssertionError: assert {'attrs': [{'...TestRoom'}]}]} == {'attrs': [{'a...TestRoom'}]}]}   Differing items:   {'attrs': [{'attrName': 'boolean', 'types': [{'entities': [{...}], 'entityType': 'TestRoom'}]}, {'attrName': 'intensit... 'entityType': 'TestRoom'}]}, {'attrName': 'temperature', 'types': [{'entities': [{...}], 'entityType': 'TestRoom'}]}]} != {'attrs': [{'attrName': 'boolean', 'types': [{'entities': [{...}], 'entityType': 'TestRoom'}]}, {'attrName': 'intensit... 'entityType': 'TestRoom'}]}, {'attrName': 'temperature', 'types': [{'entities': [{...}], 'entityType': 'TestRoom'}]}]}   Use -v to get the full diff
reporter_dataset_different_types = None, service = 't1'

    def test_aggregation_different_types_crate(
            reporter_dataset_different_types, service='t1'):
        attrs = 'temperature,intensity,boolean'
        query_params = {
            'attrs': attrs,
            'aggrMethod': 'min',
            'type': entity_type
        }
        # crate supports min on boolean
        r = query(params=query_params, service=service)
        assert r.status_code == 200, r.text
    
        obtained = r.json()
        assert isinstance(obtained, dict)
    
        expected_attrs = [
            {
                'attrName': 'boolean',
                'types':
                [{
                    'entities':
                    [{
                        'entityId': entity_id,
                        'index': ['', ''],
                        'values': [False]
                    }],
                    'entityType': entity_type
                }]
            },
            {
                'attrName': 'intensity',
                'types':
                [{
                    'entities':
                    [{
                        'entityId': entity_id,
                        'index': ['', ''],
                        'values': ['str1']
                    }],
                    'entityType': entity_type
                }]
            },
            {
                'attrName': 'temperature',
                'types':
                [{
                    'entities':
                    [{
                        'entityId': entity_id,
                        'index': ['', ''],
                        'values': [0.0]
                    }],
                    'entityType': entity_type
                }]
            },
        ]
    
        expected = {
            'attrs': expected_attrs
        }
    
>       assert obtained == expected
E       AssertionError: assert {'attrs': [{'...TestRoom'}]}]} == {'attrs': [{'a...TestRoom'}]}]}
E         Differing items:
E         {'attrs': [{'attrName': 'boolean', 'types': [{'entities': [{...}], 'entityType': 'TestRoom'}]}, {'attrName': 'intensit... 'entityType': 'TestRoom'}]}, {'attrName': 'temperature', 'types': [{'entities': [{...}], 'entityType': 'TestRoom'}]}]} != {'attrs': [{'attrName': 'boolean', 'types': [{'entities': [{...}], 'entityType': 'TestRoom'}]}, {'attrName': 'intensit... 'entityType': 'TestRoom'}]}, {'attrName': 'temperature', 'types': [{'entities': [{...}], 'entityType': 'TestRoom'}]}]}
E         Use -v to get the full diff

src/reporter/tests/test_different_types.py:229: AssertionError
```

**To Reproduce**
N/A is random... it mostly depends on assigned resource to crate.

**Expected behavior**
test passes :)

**Environment (please complete the following information):**
N/A

**Additional context**
N/A

--- a/RELEASE_NOTES.md
+++ b/RELEASE_NOTES.md
@@ -18,6 +18,7 @@
 - Updated PG8000 to 1.23.0 (#586)
 - Added request to update release notes to the pull request template (#585)
 - Added codeql-analysis workflow (#588)
+- Added wait for insert in instanceId and aggregation tests (#588)
 
 ### Bug fixes
 

--- a/src/reporter/tests/test_aggregation_different_attribute_types.py
+++ b/src/reporter/tests/test_aggregation_different_attribute_types.py
@@ -1,12 +1,13 @@
 from conftest import QL_URL
 from datetime import datetime
-from reporter.tests.utils import delete_test_data, insert_test_data_different_types
+from reporter.tests.utils import delete_test_data, \
+    insert_test_data_different_types, wait_for_insert
 import pytest
 import requests
 import dateutil.parser
 import time
 
-entity_type = "TestRoom"
+entity_type = "TestRoomAggregationDifferentTypes"
 entity_id = "TestRoom1"
 n_days = 4
 
@@ -28,7 +29,7 @@ def query(values=False, params=None, service=None):
 
 
 @pytest.fixture(scope='module')
-def reporter_dataset_different_types():
+def reporter_dataset_different_attribute_types():
     for service in services:
         insert_test_data_different_types(
             service,
@@ -36,14 +37,15 @@ def reporter_dataset_different_types():
             n_entities=1,
             index_size=4,
             entity_id=entity_id)
+        wait_for_insert([entity_type], service, 4)
 
     yield
     for service in services:
         delete_test_data(service, [entity_type])
 
 
-def test_aggregation_different_types_timescale(
-        reporter_dataset_different_types, service='t2'):
+def test_aggregation_on_different_attribute_types_timescale(
+        reporter_dataset_different_attribute_types, service='t2'):
     attrs = 'temperature,intensity,boolean'
     query_params = {
         'attrs': attrs,
@@ -165,8 +167,8 @@ def test_aggregation_different_types_timescale(
     }
 
 
-def test_aggregation_different_types_crate(
-        reporter_dataset_different_types, service='t1'):
+def test_aggregation_on_different_data_types_crate(
+        reporter_dataset_different_attribute_types, service='t1'):
     attrs = 'temperature,intensity,boolean'
     query_params = {
         'attrs': attrs,

--- a/src/reporter/tests/test_instanceId.py
+++ b/src/reporter/tests/test_instanceId.py
@@ -1,7 +1,8 @@
 from exceptions.exceptions import NGSIUsageError, InvalidParameterValue
 from conftest import QL_URL
 from datetime import datetime
-from reporter.tests.utils import insert_test_data, delete_test_data
+from reporter.tests.utils import insert_test_data, delete_test_data, \
+    wait_for_insert
 from utils.tests.common import assert_equal_time_index_arrays
 from translators.factory import translator_for
 import pytest
@@ -29,6 +30,7 @@ def reporter_dataset():
     for service in services:
         insert_test_data(service, [entity_type], n_entities=3,
                          entity_id=entity_id, index_size=n_days)
+        wait_for_insert([entity_type], service, 3)
     yield
     for service in services:
         delete_test_data(service, [entity_type])

